### PR TITLE
Changes/updating matrix files

### PR DESCRIPTION
**Pull request by igorescobar**
This PR addresses the #19 

- [x] Using the latest matrix files.

I will tackle the support of the `testmatrix_variationId` on issue #20

### CODE REVIEW

**Comment by igorescobar** [comment]
To be able to update to the latest matrix files we had to fix the way we interpret `nil` identifiers and `##null##` identifiers. They have different semantic meanings and I had to fix it. 

Discussion:
https://configcat-community.slack.com/archives/C01CN2YNWGK/p1603123573012400

File: lib/config_cat/user.ex [hunk 1]
```diff
@@ -28,7 +28,6 @@ defmodule ConfigCat.User do
   end
 
   defp normalize(attribute) do
-    attribute
-    |> to_string()
+    to_string(attribute)
```
Comment (randycoulman) [inline lib/config_cat/user.ex:31]: #minor I think we could eliminate the `normalize` function entirely and inline the `to_string` call in the one place it's called from, but not a big deal.

File: test/fixtures/testmatrix_sensitive.csv [hunk 2]
```diff
@@ -5,4 +5,4 @@ Kutya;;;;Allat;ToAll
 Sas;;;;ToAll;Kigyo
 Kutya;macska@example.com;;;Macska;ToAll
 id1;;Scotland;;Britt;Kigyo
-Macska;;USA;;ToAll;Ireland
+Macska;;USA;;ToAll;Ireland
```
Comment (randycoulman) [inline test/fixtures/testmatrix_sensitive.csv:8]: #minor Accidental change?

**Review by randycoulman — APPROVED** [review]
LGTM!

File: test/fixtures/testmatrix_sensitive.csv (hunk 2 above)
Comment (igorescobar) [reply test/fixtures/testmatrix_sensitive.csv:8]: I downloaded the file directly from their repo 👍

**Pull request merged**